project/src/main/scala/apps/analytics/AnalyticsOntology.scala

```diff
@@ -0,0 +1,75 @@
+package apps.analytics
+
+import java.io.File
+import java.net.URI
+
+import org.semanticweb.owlapi.model.{ IRI, OWLOntology, OWLOntologyManager }
+import org.semanticweb.owlapi.apibinding.OWLManager
+
+object AnalyticsOntology
```

Comment:
Still need to document this object. However, I moved all the logic related to the ontology itself here. It allows for users to load locally or remotely. If loading locally, a default option is available that is with respect to this repository layout. 

TODO: Check to make sure that the IRI of the loaded ontology matches the published IRI of our analytics ontology.